docs/_sidebar.md

```diff
@@ -1,4 +1,11 @@
 - [Home](/)
-  - [Polaris Overview](?id=Polaris-Overview)
+- [Quick Start](/quickstart/)
+- [Authentication](authentication)
+- [Database integration](databases)
 - [DevOps](/devops/)
+- [Internationalisation](internationalisation)
 - [Contribute to Polaris](/contributing/)
+- [ADRs](/ADRs/)
```

Comment:
Might be worth leaving Quick Start at the top but then putting these into alphabetical order